src/test/scala/mesosphere/marathon/core/matcher/manager/impl/OfferMatcherManagerActorTest.scala

```diff
@@ -195,6 +195,21 @@ class OfferMatcherManagerActorTest extends AkkaUnitTest with Eventually {
       Then("offer-1 is declined, since the actor did not respond in time")
       offerMatch1.future.futureValue.opsWithSource should be('empty)
     }
+
+    "not send offers to matchers that are not interested" in new Fixture {
```

Comment:
I think we also need a test which checks the other case when matchers are interested in the offer.

Reply:
agreed, added another one